src/contexts/StateContextProvider.tsx

```diff
@@ -274,6 +284,20 @@ const StateContextProvider = ({children}: StateContextProviderProps) => {
                 setFileName(args.fileName);
                 setNumEvents(args.numEvents);
                 setOnDiskFileSizeInBytes(args.onDiskFileSizeInBytes);
+                if (getConfig(CONFIG_KEY.DECODER_OPTIONS).formatString ===
+                    CONFIG_DEFAULT[CONFIG_KEY.DECODER_OPTIONS].formatString) {
+                    postPopUp({
+                        button: {
+                            title: "Open Settings",
+                            callback: () => { setIsSettingsModalOpen(true); },
+                        },
```

Comment:
Following the type change in `src/typings/notifications.ts`, how about
```suggestion
                        primaryAction: {
                            children: "Settings",
                            startDecorator: <SettingsOutlinedIcon/>,
                            onClick: () => { setIsSettingsModalOpen(true); },
                        },
```